docs/can-guides/topics/forms.md

```diff
@@ -29,17 +29,17 @@ CanJS provides many useful utilities and techniques to help with these tasks. Th
 * indentifying and navigating [guides/forms#Bindingconflicts bindings conflicts]
 * working with [guides/forms#Workingwithrelateddata related data], [guides/forms#Usingpromises promises], and [guides/forms#Resettingaselectionwhenitsparentlistchanges parent-child relationships]
 
-We recommend reading the [guides/forms#Bindings Bindings overview] section first before jumping to the topic that interests you &mdash; or start at the top to become an expert on it all.
+We recommend reading the [guides/forms#Bindings Bindings overview] section first before jumping to the topic that interests you—or start at the top to become an expert on it all.
 
 ## Bindings
 
-When working with form elements, there are two basic kinds of bindings &mdash; Event bindings and Attribute bindings. Event bindings allow you to respond to DOM [https://developer.mozilla.org/en-US/docs/Web/Events Events] and attribute bindings allow you to bind the value of HTML [https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes Attributes] to properties in your [can-view-scope scope].
+When working with form elements, there are two basic kinds of bindings—_event bindings_ and _attribute bindings_. Event bindings allow you to respond to [https://developer.mozilla.org/en-US/docs/Web/Events DOM Events] and attribute bindings allow you to bind the value of [https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes HTML Attributes] to properties in your [can-view-scope scope].
 
-The following sections will explain how to use each of these bindings on their own as well as how to combine them in more advanced binding formats.
+The following sections will explain how to use each of these bindings on their own, as well as how to combine them in more advanced binding formats.
 
 ### Event binding
 
-Event bindings allow you to respond to DOM Events. To set up an event binding, add an attribute to your element like `on:DOM_EVENT="CALL_EXPRESSION"` where `DOM_EVENT` is the event you want to respond to and `CALL_EXPRESSION` is a [can-stache/expressions/call Call Expression] for the function you want to call when the event occurs. Here is a simple example:
+Event bindings allow you to respond to [https://developer.mozilla.org/en-US/docs/Web/Events DOM Events]. To set up an event binding, add an attribute to your element like `on:DOM_EVENT="CALL_EXPRESSION"` where `DOM_EVENT` is the event you want to respond to and `CALL_EXPRESSION` is a [can-stache/expressions/call Call Expression] for the function you want to call when the event occurs. Here is a simple example:
 
 @demo demos/forms/bindings-event.html
```

Comment:
This demo uses `innerHTML` instead of having a `count` property on a view-model and rendering it with `{{count}}`. That stuck out to me and I wonder if it’ll be jarring to other people too.

Reply:
I did it this way to use as little stache as possible, so that the event binding was the only thing being introduced. You're right, that might make it more confusing. I'm open to suggestions.

Reply:
Yeah, that makes sense. I’ve updated the example to use `{{count}}`; it’s about the same amount of code and I think it makes it more consistent with the other examples.